src/map/pc.c

```diff
@@ -10567,6 +10569,9 @@ static int pc_unequipitem(struct map_session_data *sd, int n, int flag)
 		pc->unequipitem(sd, sd->equip_index[EQI_AMMO], PCUNEQUIPITEM_FORCE);
 #endif
 
+	if (sd->inventory_data[n]->type == IT_AMMO && (sd->inventory_data[n]->nameid != ITEMID_SILVER_BULLET || sd->inventory_data[n]->nameid != ITEMID_SANCTIFIED_BULLET || sd->inventory_data[n]->nameid != ITEMID_SILVER_BULLET_))
```

Comment:
missing check for ``sd->inventory_data[n] != NULL``

Reply:
oh yeah i never noticed it was outside the `sd->inventory_data[n] != NULL`